src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -1480,6 +1480,226 @@ public new abstract class Arm64 : AdvSimd.Arm64
         /// </summary>
         public static unsafe Vector<ulong> LoadVectorUInt32ZeroExtendToUInt64(Vector<ulong> mask, uint* address) => LoadVectorUInt32ZeroExtendToUInt64(mask, address);
 
+
+        ///  LoadVectorx2 : Load two-element tuples into two vectors
+
+        /// <summary>
+        /// svuint8x2_t svld2[_u8](svbool_t pg, const uint8_t *base)
+        ///   LD2B {Zresult0.B, Zresult1.B}, Pg/Z, [Xarray, Xindex]
+        ///   LD2B {Zresult0.B, Zresult1.B}, Pg/Z, [Xbase, #0, MUL VL]
```

Comment:
I think this API just maps to `LD2B {Zresult0.B, Zresult1.B}, Pg/Z, [Xbase, #0, MUL VL]`  variant so we should remove the comment about `LD2B {Zresult0.B, Zresult1.B}, Pg/Z, [Xarray, Xindex]`. @a74nh - is that correct?

Reply:
We'll could use the second if `Xindex` was the zero register. But, yes, makes sense to use `[Xbase, #0, MUL VL]`